client/css/style.css

```diff
@@ -815,25 +815,30 @@ button {
 #chat .time,
 #chat .from,
 #chat .text {
-	display: table-cell;
+	display: block;
 	padding: 2px 0;
-	vertical-align: top;
+	flex: 0 0 auto;
 }
 
 #chat .time {
 	color: #ddd;
 	text-align: right;
-	max-width: 46px;
-	min-width: 46px;
+	width: 46px;
 }
 
 #chat .from {
 	border-right: 1px solid #f6f6f6;
 	color: #b1c3ce;
 	padding-right: 10px;
 	text-align: right;
-	max-width: 134px;
-	min-width: 134px;
+	width: 134px;
+	align-self: stretch;
+}
+
+#chat .text {
+	margin: auto;
```

Comment:
I have been trying to fix this and none of `align-items: flex-start;`, `align-content: flex-start;` or `self-align: flex-start;` did the trick.
I guess I'll just remove `margin: auto` unless someone has reasons to believe this is going to break something else.